cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -287,7 +323,7 @@ install() {
     # Potentially-late-bound version of env_script_path to write to rcfiles like $HOME/.profile
     local _env_script_path_expr
 
-{% if install_path.kind == "CargoHome" %}
+if [ "$INSTALL_PATH_STRATEGY" = "CargoHome" ]; then
```

Comment:
I'm not a huge fan of us unconditionally adding chunks of likely-dead-code to each install script.